app/test/analyzer/handlers_test.dart

```diff
@@ -0,0 +1,178 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+library pub_dartlang_org.handlers_test;
+
+import 'dart:async';
+import 'dart:isolate';
+
+import 'package:gcloud/db.dart';
+import 'package:test/test.dart';
+
+import 'package:pub_dartlang_org/shared/analyzer_service.dart';
+
+import 'package:pub_dartlang_org/analyzer/backend.dart';
+import 'package:pub_dartlang_org/analyzer/models.dart';
+import 'package:pub_dartlang_org/shared/task_client.dart';
+import 'package:pub_dartlang_org/shared/task_scheduler.dart' show Task;
+
+import '../shared/handlers_test_utils.dart';
+import '../shared/utils.dart';
+
+import 'handlers_test_utils.dart';
+
+void main() {
+  group('handlers', () {
+    group('not found', () {
+      scopedTest('/xxx', () async {
+        expectNotFoundResponse(await issueGet('/xxx'));
```

Comment:
Good point, calling `await` on it. I'm not sure why this was working, maybe because it was a scoped test and it is running in a zone? Btw. the frontend's handler tests are similar.